bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/Base/_SIGMessage.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enocean.internal.eep.Base;
+
+import org.openhab.binding.enocean.internal.eep.EEP;
+import org.openhab.binding.enocean.internal.messages.ERP1Message;
+
+/**
+ *
+ * @author Daniel Weber - Initial contribution
+ */
+public abstract class _SIGMessage extends EEP {
```

Comment:
According to our codestyle class names have to match '^[A-Z][a-zA-Z0-9]*$'. Is there any reason why this cannot be applied here?

Reply:
Sorry I was not aware of this codestyle. The most common EnOcean messages are based on RPS, 1BS, 4BS and VLD messages. So I have choosen _RPSMessage, _1BSMessage, _4BSMessage and _VLDMessage for my base classes as it is not allowed to start a class name with a numeric value. As this _SIGMessage class is also a base class for certain messages I just used the same naming schema. 
For auto completion reasons in my IDE I would like to keep the differentiating part in first place of the class name.